src/Tools/dotnet-trace/CommandLine/Commands/ConvertCommand.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Tools.RuntimeClient;
+using System;
+using System.IO;
+using System.CommandLine;
+using System.CommandLine.Builder;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Tools.Trace
+{
+    internal static class ConvertCommandHandler
+    {
+        public static int ConvertFile(IConsole console, FileInfo inputFilename, TraceFileFormat format, FileInfo output)
+        {
+            try
+            {
+                if (format == TraceFileFormat.netperf)
+                    throw new ArgumentException("Cannot convert to netperf format.");
+                
+                if (!inputFilename.Exists)
+                    throw new FileNotFoundException($"File '{inputFilename}' does not exist.");
+                
+                if (output == null)
+                    output = inputFilename;
+
+                TraceFileFormatConverter.ConvertToFormat(format, inputFilename.FullName, output.FullName);
+
+                return 0;
+            }
+            catch (Exception ex)
+            {
+                Console.Error.WriteLine($"[ERROR] {ex.ToString()}");
```

Comment:
`CommandLineBuilderExtensions.UseExceptionHandler` (which is included in `CommandLineBuilderExtensions.UseDefaults`) can handle this for you.